src/test/kotlin/com/bardsoftware/backend/fleet/rmarkdown/PubsubTest.kt

```diff
@@ -39,14 +40,23 @@ class PubsubTest {
 
     @Test
     fun processFileUnzipTest() {
-        val message = "test message"
+        val rootUrl = DockerTest::class.java.getResource("/$rootFileName") ?: throw IOException()
```

Comment:
Any sense to move this code to CmdLineClient, so that we could just zip a directory?

Reply:
Done